### ISSUE TITLE
Strange exception after swapping out OmniSharp for dev version

### ISSUE THREAD

**Issue by GuardRex** [Microsoft/vscode#3920]
Moving from https://github.com/OmniSharp/omnisharp-roslyn/issues/420

After compiling and putting `omnisharp-roslyn-dev` `OmniSharp.exe` in place in order to get VS Code working happily with `dotnet cli` right now ...
```json
"csharp.omnisharp": 
    "C:\\omnisharp-roslyn-dev\\artifacts\\publish\\OmniSharp\\dnxcore50\\OmniSharp.exe",
```
I'm getting this VS Code issue ...

![capture](https://cloud.githubusercontent.com/assets/1622880/13609556/4df0e678-e50e-11e5-8e55-cfda89f70eae.PNG)

The replaced OmniSharp seems to be working ok ... IntelliSense and CodeLens seem ok. The exception seems to just be an artifact of replacing the default OmniSharp.

Anything to be concerned about? Should I just ignore this until updated VS Code tooling is released and I can drop the dev repo version?

**Comment by joaomoreno** [comment]
I apologise for the terrible way of display an error message on our part...

Can you somehow get the whole message? Triple click in there and <kbd>Ctrl C</kbd> will let you copy that message.

**Comment by GuardRex** [comment]
@joaomoreno Thanks - I was able to fix this. the entire exception is ...

c:/Program Files (x86)/Microsoft VS Code/resources/app/extensions/jrieken.vscode-omnisharp: Cannot read file c:/Program Files (x86)/Microsoft VS Code/resources/app/extensions/jrieken.vscode-omnisharp/package.json: ENOENT: no such file or directory, open 'c:\Program Files (x86)\Microsoft VS Code\resources\app\extensions\jrieken.vscode-omnisharp\package.json'

I went into the `extensions` folder and deleted the `jrieken.vscode-omnisharp` folder. That seems to have silenced the exception. It looks like it might have been an old extension that wasn't completely removed.

**Comment by lorenpaulsen** [comment]
I just encountered the same error on version 0.10.11. When installing the new C# extension, instead of restarting after the installation was complete, I accidentally opened a second instance. Afterward, I started getting this exception. No amount of attempting to uninstall and reinstall the extension would get me out of it. I had to delete the `jrieken.vscode-omnisharp` directory, then uninstall and reinstall the extension one more time to make it happy again
